docs/bokeh/source/docs/first_steps/first_steps_9.rst

```diff
@@ -191,8 +191,8 @@ widgets on the left to change the sine wave on the right:
     ></iframe>
     </div>
 
-Find the source code for this Bokeh server app in the
-`Bokeh repository on GitHub <https://github.com/bokeh/bokeh/blob/master/examples/app/sliders.py>`_.
+Find the source code for this Bokeh server app at
+:bokeh-tree:`examples/server/app/slider.py`
```

Comment:
Thanks @RajatShenoi just one tiny nit, need a full stop at the end of the sentence:
```suggestion
:bokeh-tree:`examples/server/app/slider.py`.
```